mycore-base/src/main/java/org/mycore/common/xml/MCRURIResolver.java

```diff
@@ -1026,7 +1027,7 @@ public Source resolve(String href, String base) {
                 }
             } catch (Exception ex) {
                 LOGGER.info("MCRNotNullResolver caught exception: {}", ex.getLocalizedMessage());
-                LOGGER.debug(ex.getStackTrace());
+                LOGGER.debug(Arrays.toString(ex.getStackTrace()));
```

Comment:
I noticed this while debugging UBO, just sneaked it in for better debug logs.

Reply:
usually we log the whole exception object
`LOGGER.debug(ex, "MCRNotNullResolver caught exception")`